algebra-jackson/src/test/java/com/hubspot/algebra/ResultModuleTest.java

```diff
@@ -341,6 +357,42 @@ public void itDeserializesNullErr() throws Exception {
     );
   }
 
+  @Test
+  public void itDeserializesNestedOkOk() throws Exception {
```

Comment:
Should there also be serialization tests?

Reply:
:+1: